variables.tf

```diff
@@ -77,15 +93,15 @@ variable "esxi_size" {
 }
 
 variable "facility" {
-  default = "dfw2"
+  default = "ny5"
 }
 
 variable "router_os" {
   default = "ubuntu_18_04"
 }
 
 variable "vmware_os" {
-  default = "vmware_esxi_7_0"
+  default = "vmware_esxi_6_7"
```

Comment:
Is there a reason we would want to downgrade the VMWare release used in this module? The vSphere module is known to work with 7.0.

I think we should keep this at 7.0 and let the Anthos module set it to 6.7, if needed.

Reply:
Agreed. This should be 7.0 I just had a 6.7 vCenter ISO handy... Should have changed this in my terraform.tfvars instead.